db/fixtures/miq_product_features.yml

```diff
@@ -2570,20 +2570,12 @@
     :feature_type: view
     :identifier: tasks_view
     :children:
-    - :name: All VM and Container Analysis Tasks
-      :description: Display Lists of All VM and Container Analysis Tasks
-      :feature_type: view
-      :identifier: job_all_smartproxy
```

Comment:
Nice.

However, I remember there was some anomalies in the ops ui once you remove a Feature that is already assigned to existing roles.

I think the reproducer was to: try to copy a user-role to new user-role, then remove some features and edit the role permissions.

The safest way is to never release the unused features.

Hence, I would vote for fine/yes + blocker to ensure this is never released.

Thoughts?